packages/error-boundary-fallback/README.md

```diff
@@ -0,0 +1,24 @@
+# Errorboundary Fallback
+
+This package provides a fallback component which you can use as an error boundary fallback.
+
+## Installation
+
+```node
+yarn add @opensrp/error-boundary-fallback
+```
+
+#### Code example
+
+```javascript
+import * as Sentry from '@sentry/react';
+import App from '..';
+import { ErrorBoundary } from '@opensrp/error-boundary-fallback';
+
+ReactDOM.render(
+  <ErrorBoundary dsn={SENTRY_DSN} homeUrl={HOME_URL}>
```

Comment:
Could we enter hard coded values for `dsn` and `homeUrl` attributes? I think a person reading the README might wonder what is `SENTRY_DSN` and `HOME_URL`

Reply:
Also, i think its best if we add in the README what each prop does i.e dsn and homeUrl props